packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -82,28 +83,37 @@ export function useGasSummary(
   })
 
   // Estimated L1 gas fees, denominated in Ether, represented as a floating point number
-  const estimatedL1GasFees = useMemo(
-    () => parseFloat(utils.formatEther(result.estimatedL1Gas.mul(l1GasPrice))),
-    [result.estimatedL1Gas, l1GasPrice]
-  )
+  const estimatedL1GasFees = useMemo(() => {
+    // The relayer pays the gas fees
+    if (isSmartContractWallet) {
+      return 0
+    }
```

Comment:
just a comment: currently the gas estimation on the summary does not explain this for smart contract wallet so it is not informative at all to have the estimated L1 gas row on the summary UI

Reply:
You're right, I've moved it back to just for disabled and error message logic. In the next PR I'll try to abstract it into a nice little function